pass uid and gid as build argument

ap-wtioit opened the pull request:
Set the UID and GID for building scaffolding

build args implemented in https://github.com/Tecnativa/doodba/pull/224

ap-wtioit committed:
[DCK] Pass uid and gid as build argument

This add local support for https://github.com/Tecnativa/doodba/pull/224, allowing to fix permissions mismatch between odoo user in the container and the developer user in the host computer.

ap-wtioit commented:
force pushed to reset the author to my company user

yajo commented:
Thanks, it seems like you nailed it this time! :wink: 

I set as WIP until we finish  Tecnativa/doodba#224.

yajo committed:
[VSCODE] Support UID/GID matching by default

Built-in tasks that actually user the `$UID` and `$GID` variables to avoid permission problems while developing now include these in the task definition itself.

This provides a smooth out-of-the-box experience.

The "Update git code" task already did this, but printed some warnings that are now removed. It has also been moved to the "build" group, since updating git code can be considered somehow a "build" mechanism (as oppposed to the rest of tasks which are more generic). These kind of tasks are reachable by a quick specific command in VSCode.

yajo commented:
I took the freedom to amend your commit message, to make it similar to the rest in the project (I tend to prefix all of them with `[DCK]`  so you can easily distinguish them in your private forks).

I also added another one with out-of-the-box support in vscode tasks.

Let's wait for the CI.
